python/hail/expr/types.py

```diff
@@ -166,14 +166,11 @@ def _convert_to_py(self, annotation):
         return annotation
 
     def _convert_to_j(self, annotation):
-        if annotation is not None:
-            return Env.jutils().makeLong(annotation)
-        else:
-            return None
+        raise NotImplementedError('int64 conversion from Python to JVM')
```

Comment:
I'm slightly confused by this (and the _convert_to_j for TInt32).  Why do we need to do anything for this?  Not being implemented certainly seems wrong.

Reply:
they're both a mess and can hopefully go away soon.